pkg/download/classic/filter.go

```diff
@@ -40,7 +40,7 @@ var apiContentFilters = map[string]contentFilter{
 			if json["dashboardMetadata"] != nil {
 				metadata := json["dashboardMetadata"].(map[string]interface{})
 
-				if metadata["preset"] != nil && metadata["preset"] == true {
+				if metadata["preset"] != nil && metadata["preset"] == true && metadata["owner"] == "Dynatrace" {
```

Comment:
So can non-Dynatrace presets be edited/modified?
Or are all presets 'create-only' rather than fully unmodifiable? If that's the case we should probably skip them on deploy if they exist on the environment, rather than ignore on download 🤔

Reply:
as far as I can see, user-defined preset dashboards are just as modifiable as non-preset dashboards

Reply:
One more stupid question: are Dynatrace presets actually unmodifable on the API? Iirc this filter is quite old and might have been based on their immutability in the UI only

Reply:
thx for checking